src/workerd/api/container.c++

```diff
@@ -30,6 +30,30 @@ void Container::start(jsg::Lock& js, jsg::Optional<StartupOptions> maybeOptions)
   }
   req.setEnableInternet(options.enableInternet);
 
+  KJ_IF_SOME(env, options.env) {
+    auto list = req.initEnvironmentVariables(env.fields.size());
+    for (auto i: kj::indices(env.fields)) {
+      auto field = &env.fields[i];
+      JSG_REQUIRE(!field->name.contains("="), Error,
```

Comment:
`contains` looks for a substring, but since it's just one character, I would use `findFirst`.

```suggestion
      JSG_REQUIRE(field->name.findFirst('=') == kj::none, Error,
```